docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -89,6 +89,11 @@
           helm_chart_overrides: "/home/{{ ansible_ssh_user }}/{{ helm_chart_overrides | basename }}"
         when: helm_chart_overrides is defined
 
+      - set_fact:
+          helm_chart_overrides: "/usr/local/share/applications/overrides/wind-river-cloud-platform-deployment-manager-overrides-subcloud.yaml"
+          manager_chart: "/usr/local/share/applications/helm/wind-river-cloud-platform-deployment-manager-*.tgz"
+        when: ("subcloud" in get_distributed_cloud_role.stdout and "not uploaded" in user_uploaded_artifacts)
```

Comment:
Note: the user_uploaded_artifacts may not be defined, and the playbook will throw out an error in this case. Need to test this playbook on standalone system

Reply:
run the playbook without user_uploaded_artifacts

<img width="741" alt="image" src="https://github.com/Wind-River/cloud-platform-deployment-manager/assets/129255520/96fe3e2f-579a-4034-a944-1f00fcb4efff">

Reply:
executed the playbook with user_uploaded_artifacts defined to "not uploaded"

<img width="550" alt="image" src="https://github.com/Wind-River/cloud-platform-deployment-manager/assets/129255520/7f2194fe-39d4-4274-a3a5-cba9fe074848">


<img width="746" alt="image" src="https://github.com/Wind-River/cloud-platform-deployment-manager/assets/129255520/fdbd0004-c747-497d-9299-8f2b09509ab3">

Reply:
Thanks for the re-test, we can hold this PR until the change in the dc repo finalized

Reply:
And the "dcmanager subcloud reconfig" test is required once the change in DC repo finalized

Reply:
As discussed in scrum, playbook changes can be merged independent of the dc repo.